finality-aleph/src/network/substrate.rs

```diff
@@ -258,12 +338,13 @@ impl<B: Block, H: ExHashT> RawNetwork for SubstrateNetwork<B, H> {
         protocol: Protocol,
     ) -> Result<Self::NetworkSender, Self::SenderError> {
         Ok(SubstrateNetworkSender {
-            // Currently method `notification_sender` does not distinguish whether we are not connected to the peer
-            // or there is no such protocol so we need to have this worthless `SenderError::CannotCreateSender` error here
-            notification_sender: self
-                .network
-                .notification_sender(peer_id, self.naming.protocol_name(&protocol))
-                .map_err(|_| SenderError::CannotCreateSender(peer_id, protocol))?,
+            message_sink: match protocol {
```

Comment:
If you remove the `_network` field then this trait shouldn't be necessary at all, right? In particular, we never call this method, or am I wrong?

Reply:
removed